### ISSUE TITLE
WCAG 2.0 2.1.1: Keyboard functionality and CKEditor 5's toolbar

### ISSUE THREAD

**Issue by spudthebud** [ckeditor/ckeditor5#15164]
In CK Editor 5, the toolbar seems to require the user to use the mouse.
It is not keyboard accessible. This is a requirement per WCAG 2.0 [success criteria 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## 📝 Provide detailed reproduction steps (if any)

1. Go to the [CK Editor 5 demo](https://ckeditor.com/ckeditor-5/demo/feature-rich/)
2. Use the mouse to put the focus in the editor
3. Press SHIFT + TAB to try to go to a control type in the toolbar.
4. Notice how the keyboard focus leaps over the toolbar
5. Press TAB to go into the control type in the toolbar
6. Notice tow the keyboard focus leaps over the toolbar and goes into the edit.

### ✔️ Expected result
The focus should go to a control type in in the toolbar.
 This happens with GitHub's editor. 

### ❌ Actual result

CK Editor skips the toolbar

## 📃 Other details

* Browser:   Chrome
* OS: Windows 10
* First affected CKEditor version: 5
* Installed CKEditor plugins: ?

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mgifford** [comment]
I'm trying to replicate this in the demo here [https://ckeditor.com/ckeditor-5/demo/feature-rich/](https://ckeditor.com/ckeditor-5/demo/feature-rich/)  
  
Definitely needs a lot of work for keyboard only accessibility.

**Comment by Witoso** [comment]
Thanks for the report!

To focus a toolbar, main one, or when you're on the element that has its toolbar, e.g., image, the <kbd>ALT</kbd>+<kbd>F10</kbd>. We need to do a a better job in documenting it.

**Comment by spudthebud** [comment]
Are there any other unique commands you can think of that we should be conveying to our colleagues?

**Comment by Witoso** [comment]
That's the main one. @oleq is there anything els regarding the UI. @godai78 what could be the best place in the docs to mention this.

**Comment by godai78** [comment]
Not sure, but I would suggest putting it both in the keyboard shortcuts guide and probably a mention in the toolbar guide.

**Comment by oleq** [comment]
It's on our to-do list https://github.com/ckeditor/ckeditor5/issues/1014.

**Comment by godai78** [comment]
After a second check, this keyboard shortcut is actually documented already in the keyboard shortcut guide: https://ckeditor.com/docs/ckeditor5/latest/features/keyboard-support.html#user-interface-and-navigation

It's more a question of discoverability at this stage.

**Comment by lnmp4000** [comment]
Default key strokes on Ubuntu linux for `Alt`+`F10` maximise the window. This prevents access to the CK Editor toolbar instead it just toggles the window size.

Alt+F10 or Super+↑

[Maximize](https://help.ubuntu.com/stable/ubuntu-help/shell-windows-maximize.html.en) a window. Press Alt+F10 or Super+↓ to restore a maximized window to its original size.

https://help.ubuntu.com/stable/ubuntu-help/keyboard-nav.html.en#:~:text=Use%20Alt%20%2B%20F10%20to%20maximize,F10%20both%20maximizes%20and%20restores.&text=Move%20the%20current%20window.,keys%20to%20move%20the%20window.

**Comment by jannakha-copy-01** [comment]
this issue is outdated
#1014 has been implemented
new issue created ##16048 for discoverability of shortcuts

**Comment by Witoso** [comment]
Ok, let's move to #16048.
